collections-lists/ConvertIAsyncEnumerableToList/ConvertIAsyncEnumerableToListTests/IAsyncEnumerableToListTests.cs

```diff
@@ -0,0 +1,78 @@
+namespace ConvertIAsyncEnumerableToListTests;
+
+public class IAsyncEnumerableToListTests
+{
+    private static readonly CancellationToken CancellationToken = new CancellationToken();
+    private static readonly CancellationTokenSource CancellationTokenSource = new CancellationTokenSource();
```

Comment:
We should move these to be local to the tests that will use them. That way there is no issue of calling cancel in one test and it being respected in another test where we don't actually want a cancellation. It also makes the tests cleaner and more straightforward.